### PR TITLE
Step 7 in release 0.1.21 (changelog)

### PR DESCRIPTION
**Pull request by jingzhang36**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1478)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,1200 +1,8 @@
 # Change Log
 
-## [0.1.20](https://github.com/kubeflow/pipelines/tree/0.1.20) (2019-05-14)
-[Full Changelog](https://github.com/kubeflow/pipelines/compare/0.1.19...0.1.20)
+## [0.1.13](https://github.com/jingzhang36/pipelines/tree/0.1.13) (2019-03-22)
```
Comment (gaoning777) [inline CHANGELOG.md]: Weird, it is supposed to generate changelog for 0.1.21.

Comment (Ark-kun) [reply CHANGELOG.md]: Also the repo is wrong

**Comment by Ark-kun** [comment]
Looks like the new changelog is empty.

File: CHANGELOG.md (hunk 1 above)
Comment (jingzhang36) [reply CHANGELOG.md]: Might be because my local master branch is messed up :(

**Comment by jingzhang36** [comment]
I'll close this and try again with the upstream clone

**Comment by jingzhang36** [comment]
I have a new PR now https://github.com/kubeflow/pipelines/pull/1527/. With the correct version 0.1.21 and in the upstream repo (instead of my own fork repo)

**Pull request closed without merging**